src/components/Rewards/withdraw.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react'
+import { withdrawLP } from "../../hooks/Rewards";
+import styled from 'styled-components'
+import { useActiveWeb3React } from '../../hooks'
+import { ButtonPrimary } from '../Button';
+import EstimatedRewards from './estimatedRewards';
+const Wrapper = styled('div')`
+text-align:left;
+display: flex;
+flex-wrap: wrap;
+`
+
+const Wrapper2 = styled('div')`
+display: flex;
+flex: wrap;
+padding-bottom: 14px;
+margin: auto;
+margin-top: 10px;
+margin-bottom: 10px;
+  width: 100%;
+    overflow: hidden;
+    text-align: left;
+    justify-content: flex-end;
+`
+
+const PrecentageWrapper = styled('div')`
+display: flex;
+padding-left: 24px;
+padding-right: 24px;
+flex: wrap;
+padding-bottom: 24px;
+margin: auto;
+  width: 100%;
+    overflow: hidden;
+    text-align: left;
+    justify-content: flex-end;
+`
+const InputWrapper = styled('div')`
+display: flex;
+flex: wrap;
+margin: auto;
+border-radius: 12px;
+padding: 12px;
+border: 2px solid white;
+  width: 100%;
+    overflow: hidden;
+    text-align: left;
+    justify-content: flex-end;
+    >span{
+      margin: auto;
+    }
+`
+const Input = styled('input')`
+display: flex;
+flex: 1;
+font-size: 16px;
+background: none;
+border:none;
+padding: 0.5rem;
+color: white;
+:focus {
+        outline: none;
+      }
+`
+const Text = styled('div')`
+display: flex;
+justify-content: flex-end;
+padding-right: 10px;
+text-align: right;
+color: #B5B9D3;
+font-size: 14px;
+font-weight: 400;
+line-height: 18px;
+`
+
+const Balance = styled('div')`
+display: flex;
+text-align: center;
+font-size: 14px;
+font-weight: 500;
+line-height: 18px;
+
+`
+
+const Percentage = styled('button')`
+margin: 2px;
+margin-top: 10px;
+padding: 4px;
+display: inline-block;
+text-align: center;
+justify-content: flex-end;
+color: white;
+font-size: 14px;
+font-weight: 400;
+line-height: 18px;
+width: 100%;
+border-radius: 999px;
+border: solid 2px white;
+background: none;
+`
+
+export default (props: any) => {
```

Comment:
add a name for the function, add an interface to the props for the values you are extracting